Student-Grace-Calculation/src/GradeSchemaCalculator.java

```diff
@@ -35,6 +35,11 @@ public static void main(String[] args) {
 		
 		Scanner scan = new Scanner(System.in);
 		
+		String showResults = "Your provided details for Grade Schema are like: \n"+
```

Comment:
can you rename the project with name grades (all lower case and no dashes) instead of Student-Grace-Calculation